evaluation/DPVerification.py

```diff
@@ -5,12 +5,15 @@
 # i.e. passing (epsilon, delta) - DP condition
 # If the definition is not passed, there is a bug or it is a by-design bug in case of passing actual aggregates
 
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Why is this sys.path line needed? Should some of this code live in the burdock library to avoid this?

Reply:
evaluation classes are not in burdock and they're being called from the Jupyter notebook as well as the pytest in tests/evaluation folder. This is helping support both right now. May be there's a more elegant way to do this. If I remove this, I get a ModuleNotFound error, even though I have added __init__.py in evaluation folder.

Reply:
We can move to its own separate module in the future.  In the short term, you can run local code by doing python from the burdock root, and this also works for pytest (python -m pytest) from burdock root.  The Jupyter notebook can add the search path for now, since that's the only workaround I know for Jupyter before we make the module setup.